R/language.R

```diff
@@ -20,8 +20,13 @@
 #' q3 <- quote(x + 1)
 #' is_lang(q3)
 #' is_call(q3)
+#'
+#' # A literal is either a scalar atomic vector or a name:
```

Comment:
From parsing. I'm not sure it is quite right to call literal something that has been parsed and stored in a variable however, since they have become constructed objects at that point. Maybe this should just be removed.